### ISSUE TITLE
Experimenting with Pyodide / pyscript / jupyterlite

### ISSUE THREAD

**Issue by stonebig** [winpython/winpython#1077]
it's all about Python Stack inside a WASM (Web Assembly) container.

on WASM, there is two WebAssembly platform **wasm/emscripten** and a brewing standard in **wassm/wasi** :
- **wasm/emscripten**:
   - supposes the WebAssembly is running surrounded per a [Javascript motor, and so must run [inside a browser or node.js](https://github.com/python/cpython/issues/90473#issuecomment-1093941841),
      - pyodide is "designed" to run in this 'mature' state,
      - pyscript seems to suppose the same for now.   
   - **wasm/wasi**:
      - is for **W**eb**A**ssembly **S**ystem **I**nterface standalone application and runtime,
      - is an non-finalised state of  [standardisation](https://github.com/WebAssembly/WASI#webassembly-system-interface),
      - is only a [possible future option with of Python-3.11b2 at least](https://github.com/python/cpython/issues/90473),
      - would require a target standalone runtimes like [wasmtime or wasmer](https://news.ycombinator.com/item?id=27540994#:~:text=The%20impression%20that%20I%20get,that%20runs%20WASI%20binaries%20directly.):
        - wasmer is more targeted towards people who want to embed a wasm runtime inside an existing application that they're building, 
        - wasmtime is more meant to be used as a standalone program that runs WASI binaries directly.

"wasm/emscripten" is our object of experiment.
"wasm/wasi" is too pre-alpha, and [out of pyodide scope.](https://github.com/pyodide/pyodide/issues/558)

**Comment by stonebig** [comment]
first example:
- simple webpage in docs/pyscript https://github.com/winpython/winpython_afterdoc/tree/master/docs/pyscript
- shipped with Winpython64-3.11.0.0dotb1.exe in https://sourceforge.net/projects/winpython/files/WinPython_3.11/3.11.0.0/betas/

**Comment by stonebig** [comment]
now using 
```
    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
    <script defer src="https://pyscript.net/latest/pyscript.js"></script>

```
